api/activities/activities.go

```diff
@@ -7,41 +7,32 @@ import (
 	"github.com/brooklyncentral/brooklyn-cli/net"
 )
 
-func Activity(network *net.Network, activity string) models.TaskSummary {
+func Activity(network *net.Network, activity string) (models.TaskSummary, error) {
 	url := fmt.Sprintf("/v1/activities/%s", activity)
-	body, err := network.SendGetRequest(url)
-	if err != nil {
-		fmt.Println(err)
-	}
+    var task models.TaskSummary
+    body, err := network.SendGetRequest(url)
+    if err != nil {
+        return task, err
+    }
 
-	var task models.TaskSummary
 	err = json.Unmarshal(body, &task)
-	if err != nil {
-		fmt.Println(err)
-	}
-	return task
+	return task, err
 }
 
-func ActivityChildren(network *net.Network, activity string) []models.TaskSummary {
+func ActivityChildren(network *net.Network, activity string) ([]models.TaskSummary, error) {
 	url := fmt.Sprintf("/v1/activities/%s/children", activity)
-	body, err := network.SendGetRequest(url)
-	if err != nil {
-		fmt.Println(err)
-	}
+    var tasks []models.TaskSummary
+    body, err := network.SendGetRequest(url)
+    if err != nil {
+        return tasks, err
+    }
 
-	var tasks []models.TaskSummary
 	err = json.Unmarshal(body, &tasks)
-	if err != nil {
-		fmt.Println(err)
-	}
-	return tasks
+	return tasks, err
 }
 
-func ActivityStream(network *net.Network, activity, streamId string) string {
+func ActivityStream(network *net.Network, activity, streamId string) (string,  error) {
 	url := fmt.Sprintf("/v1/activities/%s/stream/%s", activity, streamId)
 	body, err := network.SendGetRequest(url)
-	if err != nil {
-		fmt.Println(err)
-	}
-	return string(body)
+	return string(body), err
```

Comment:
Need to be careful here, as it is likely that string var `body` will not have been created.  Safer to just return empty string `""`.